### DISCUSSION TITLE
Stringer multi-accounts and payments

### DISCUSSION THREAD

**Discussion by mockdeep** [stringer-rss/stringer#938] [Announcements]
@swanson @gabrielpoca @Koronen Hey all, I wanted to reach out about the future of Stringer. I have been gradually working towards allowing multiple users, which then enables me to set up a paid instance. People will still be free to set up their own instances as they like, but if they'd rather pay for hosting then I'll make that available. I'm also planning on setting up GitHub sponsors for people who do want to host their own but also want to contribute some funds.

I don't have any reason to believe much money will come in from either of these, but wanted to let all of you know in case you have thoughts or concerns. I don't know what's considered fair, but my default is to pipe the money into my own coffers as I'm the only one actively working on the project at this point. I'm happy to consider any alternatives if people have suggestions.

In terms of timeline, I expect to have multiple account support finished up this week and then I'll integrate payments next week.

Any thoughts?

**Comment by gabrielpoca** [comment]
Hey @mockdeep 👋
I'm happy to see you excited about the project, and I hope the paid instance and the sponsor generate money for you! I don't have any issues or concerns with your plan!

**Comment by swanson** [comment]
:+1:
